libs/author/website/src/lib/author.tsx

```diff
@@ -6,6 +6,7 @@ import {
 } from '@wepublish/website/builder'
 
 declare module 'react' {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Would cause an error here:
<img width="736" alt="image" src="https://github.com/wepublish/wepublish/assets/1131901/1ca0384f-212f-4872-af89-e021cef1f5db">